man/systemd.network.xml

```diff
@@ -899,6 +895,47 @@
     </refsect1>
 
   <refsect1>
+    <title>[IPv6AcceptRouterAdvertisements] Section Options</title>
```

Comment:
well, the current logic is to enable/disable a protocol in the main [Network] section, and then configure its details in a protocol-specific section. Hence there is DHCP= in [Network] for the global on/off switch for DHCP, plus a section [DHCP] configuring the details of it. Similar for DHCPServer= and [DHCPServer], Bridge= and [Bridge]. Hence, in order to follow this logic I added [IPv6AcceptRouterAdvertisements] to configure the details of the RA client that is enabled/disabled via IPV6AcceptRouterAdvertisements= from [Network].

[DHCPDUID] is a different situation. There I was arguing that it should just be part of [DHCP], and not separate.

Does this make sense?